docs/docs/APIdocs/layers/nn/Simple Layers/LookupTable.md

```diff
@@ -0,0 +1,86 @@
+## LookupTable ##
```

Comment:
The comments are from its JavaDoc.
It will randomly generate weights and select the corresponding lines of the weights.